src/control/server/mgmt_svc.go

```diff
@@ -828,7 +827,7 @@ func (svc *mgmtSvc) StopRanks(ctx context.Context, req *mgmtpb.RanksReq) (*mgmtp
 
 	select {
 	case <-stopped:
-	case <-time.After(timeout):
+	case <-time.After(svc.harness.rankReqTimeout * 2):
```

Comment:
Why double?

Reply:
this will be modified in https://github.com/daos-stack/daos/pull/1878 to be more clear with regard to intent